### PR TITLE
Add new parent term to congenital myasthenic syndrome 12

### PR DESCRIPTION
**Pull request by twhetzel**
closes https://github.com/monarch-initiative/mondo/issues/7637

Confirmed new parent suggestion from description in https://omim.org/entry/610542 and icd11 description.

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -284615,9 +284615,11 @@ xref: Orphanet:353327 {source="OMIM:610542"}
 xref: Orphanet:590 {source="OMIM:610542"}
 xref: UMLS:C3552335 {source="MONDO:equivalentTo", source="MEDGEN:765249", source="MONDO:MEDGEN"}
 is_a: MONDO:0000182 {source="DC-OMIM:610542", source="OMIM:610542"} ! congenital myasthenic syndrome with tubular aggregates
+is_a: MONDO:0024322 {source="https://orcid.org/0000-0001-6330-7526"} ! disorder of glycosylation
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:284618]: If you confirmed the SC with OMIM and ICD11, please add these x-ref as a source.

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
Please see comment about source for SC

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:284618]: added omim and pubmed references

**Review by sabrinatoro — APPROVED** [review]
I approve and merge

**Pull request merged**